src/platform/packages/shared/kbn-search-connectors/components/configuration/connector_configuration.tsx

```diff
@@ -21,7 +21,7 @@ import {
 
 import { i18n } from '@kbn/i18n';
 
-import { isDeepEqual } from 'react-use/lib/util';
+import isDeepEqual from 'fast-deep-equal/react';
```

Comment:
the internal location of that util has changed, but I decided to just use the underlying lib directly instead of an export https://github.com/streamich/react-use/blob/master/src/misc/isDeepEqual.ts